### ISSUE TITLE
V1.1.1 stopped working

### ISSUE THREAD

**Issue by lacherogwu** [otherguy/alfred-passwords-workflow#74]
I was using it and today I check and it stopped working.
 
https://user-images.githubusercontent.com/49835614/189490314-39ac48b4-f6ed-4b75-8d37-d06d6b14aa18.mov

It's weird because when I run the script using Terminal, the response looks correct

```json
{"items": [{"title": "20 character password", "subtitle": "5{s/j~CKetW]d:^v*<Zo", "valid": true, "arg": "5{s/j~CKetW]d:^v*<Zo"}, {"title": "20 character password (no special characters)", "subtitle": "I8DK50D2H7SnmVMc2mFA", "valid": true, "arg": "I8DK50D2H7SnmVMc2mFA"}, {"title": "XKCD Password (3 words)", "subtitle": "chess-deem-shrink", "valid": true, "arg": "chess-deem-shrink"}, {"title": "XKCD Password (4 words)", "subtitle": "driven-sloppily-manmade-cake", "valid": true, "arg": "driven-sloppily-manmade-cake"}]}
```
<img width="1528" alt="image" src="https://user-images.githubusercontent.com/49835614/189490434-d9d5ea96-80f0-4490-8d71-f3f605c523e2.png">

Just on the terminal looks like the output ends with %, as if it doesn't close the write stream.

Thanks!

**Comment by vitorgalvao** [comment]
As per your screenshot, the workflow is producing extra text before the JSON, making it invalid. Works fine for me, though.

**Comment by lacherogwu** [comment]
But the response looks correct

**Comment by vitorgalvao** [comment]
It looks correct, but it is not due to the extra text. That makes it invalid JSON.

**Comment by lacherogwu** [comment]
Does it work for you at the same version I’m at?

**Comment by vitorgalvao** [comment]
Yes, it works on 1.1.1, though on Alfred 5 and macOS 12.6. Shouldn’t make a difference, though. From your screenshot it seems that it was the library informing it was checking for updates which broke it. Try reinstalling the latest version.

**Comment by lacherogwu** [comment]
Will do thanks

**Comment by lacherogwu** [comment]
works thanks!
